planning_interface/move_group_interface/src/move_group.cpp

```diff
@@ -689,6 +689,31 @@ class MoveGroup::MoveGroupImpl
     }
   }
 
+  bool validatePlan(const Plan &plan)
+  {
+    robot_state::RobotStatePtr current_state;
+    if (!getCurrentState(current_state))
+        return false;
+    if (plan.trajectory_.joint_trajectory.points.empty())
+        return true;
+
+    const trajectory_msgs::JointTrajectory &trajectory = plan.trajectory_.joint_trajectory;
+    const std::vector<double> &positions = trajectory.points.front().positions;
+    size_t n = trajectory.joint_names.size();
+    if (positions.size() != n)
+        return false;
+
+    for (size_t i=0; i < n; ++i)
```

Comment:
Could you be more verbose, please. Would do you mean by std::? Where is it missing?